server/src/main/java/io/crate/auth/JWTAuthenticationMethod.java

```diff
@@ -102,7 +107,10 @@ public JWTAuthenticationMethod(Roles roles,
                 audience = jwtProperties.aud() != null ? jwtProperties.aud() : audience;
             }
 
-            JwkProvider jwkProvider = urlToJwkProvider.apply(decodedJWT.getIssuer());
+            String jwtIssuer = decodedJWT.getIssuer();
```

Comment:
Well, since we have decodedJWT.getIssuer() usage on master, it's rather issue of https://github.com/crate/crate/commit/f8bcce833ea81b9ce84e4ad2e6e21e6a3a2cde23, I will follow up on it

Reply:
https://github.com/crate/crate/pull/16959